build.gradle.kts

```diff
@@ -40,12 +40,21 @@ zapAddOn {
 
     manifest {
         author.set("KSASAN preetkaran20@gmail.com")
+
+        dependencies {
+            addOns {
+                register("commonlib")
+            }
+        }
     }
 }
 
 dependencies {
     implementation("org.json:json:20190722")
     // https://mvnrepository.com/artifact/com.nimbusds/nimbus-jose-jwt
     implementation("com.nimbusds:nimbus-jose-jwt:8.3")
-    implementation("org.zaproxy.addon:commonlib:1.0.0")
+
+    compileOnly("org.zaproxy.addon:commonlib:1.0.0")
```

Comment:
Does ZAP depends on this addon ? if not then How is this library present in ZAP

Reply:
When JWT is installed it will be installed from marketplace as a dependency.

Reply:
ok, Does "load addon from file" will also do the same operation ?

Reply:
Yup yup, already tested 🙂

I guess it’d fail if you are on a disconnected system and somehow have the addon file.